### DISCUSSION TITLE
Elección Proyecto sustituto

### DISCUSSION THREAD

**Discussion by danielapavas** [rramosp/ai4eng.v1#271] [00 - General]
Hola profe,

He estado mirando las competencias de kaggle relacionadas con el transporte y me gustaría desarrolllar esta [competencia](https://www.kaggle.com/competitions/nyc-taxi-trip-duration/overview) que trata de predecir la duración total de los viajes en taxi en la ciudad de Nueva York.  
Me podrias confirmar si puedo empezar a trabajar con esa, o debo tener en cuenta algun tipo de restricciones (como número de columnas o filas), ya que en la plataforma no se especifica.

**Comment by Felipe-RA** [comment]
Saludos @danielapavas 

Este dataset no cumple el requisito de columnas, tal como esta listado en las indicaciones del proyecto

**Reply by Felipe-RA** [reply]
Saludos nuevamente @danielapavas 

Se me pasó recordarte que los requisitos sí que se listan en la página.

Explicitamente, [aquí:](https://rramosp.github.io/ai4eng.v1/content/M00_intro_udea.html#datasets)

![image](https://github.com/rramosp/ai4eng.v1/assets/49454068/40615ea5-e1f9-4dd2-9493-5ee162dbb03b)

**Comment by danielapavas** [comment]
Hola @Felipe-RA , Me gustaria saber si esta [competencia](https://www.kaggle.com/competitions/porto-seguro-safe-driver-prediction/data) que trata de predecir la probabilidad de que el titular de una póliza de seguro de automóvil presente un reclamo, cumple con los requisitos solicitados. Muchas gracias

**Reply by Felipe-RA** [reply]
Saludos @danielapavas 

Sólo le falta el requisito de datos faltantes, lo cual puedes simular.

Hasta pronto!
